src/main/java/com/amazon/dlic/auth/http/saml/AuthTokenProcessorHandler.java

```diff
@@ -423,26 +421,16 @@ private String extractSubject(SamlResponse samlResponse) throws Exception {
         for (String role : values) {
             if (role != null) {
                 for (String splitRole : role.split(samlRolesSeparator)) {
```

Comment:
change `samlRolesSeparator` to compiled pattern.

Reply:
Done.